pkg/watermark/fetch/edge_fetcher.go

```diff
@@ -63,11 +60,10 @@ func NewEdgeFetcher(ctx context.Context, bucketName string, storeWatcher store.W
 
 	return &edgeFetcher{
 		ctx:              ctx,
-		bucketName:       bucketName,
-		storeWatcher:     storeWatcher,
+		bucketName:       manager.GetBucket(),
 		processorManager: manager,
 		lastProcessedWm:  lastProcessedWm,
-		log:              log.With("bucket", bucketName),
+		log:              log.With("bucket", manager.GetBucket()),
```

Comment:
Do we still need this if we have `log := logging.FromContext(ctx).With("bucketName", manager.GetBucket())` ??....

Reply:
removed